src/ResourceManagement/HDInsight/HDInsight.Tests/ScenarioTests/CreateClusterTests.cs

```diff
@@ -235,6 +235,7 @@ public void TestCreateLinuxClusterWithPremiumTier()
                 var resourceGroup = HDInsightManagementTestUtilities.CreateResourceGroup(resourceManagementClient);
 
                 var cluster = GetClusterSpecHelpers.GetCustomCreateParametersIaas();
+                cluster.Version = "3.4";
```

Comment:
Does this need to be a hard-coded value?  How would the user discover this value?